Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.cpp

```diff
@@ -276,6 +276,42 @@ void RemoteGraphicsContextGL::simulateEventForTesting(WebCore::GraphicsContextGL
     m_context->simulateEventForTesting(event);
 }
 
+void RemoteGraphicsContextGL::getBufferSubDataInline(uint32_t target, uint64_t offset, size_t dataSize, CompletionHandler<void(std::span<const uint8_t>)>&& completionHandler)
+{
+    assertIsCurrent(workQueue());
+    static constexpr size_t getBufferSubDataInlineSizeLimit = 64 * KB; // NOTE: when changing, change the value in RemoteGraphicsContextGLProxy too.
+    if (!dataSize) {
+        completionHandler({ });
+        return;
+    }
+    MallocPtr<uint8_t> bufferStore;
+    std::span<uint8_t> bufferData;
+    if (dataSize < getBufferSubDataInlineSizeLimit) {
```

Comment:
There's a off-by-one where dataSize == getBufferSubDataInlineSizeLimit is not good for anybody